stories/s1grdew.stories.js

```diff
@@ -266,6 +266,32 @@ function renderTilesList(containerEl, list) {
   });
 }
 
+export const findDates = () => {
+  const layer = new S1GRDAWSEULayer(instanceId, layerId);
+  const bbox = new BBox(CRS_EPSG4326, 11.9, 12.34, 42.05, 42.19);
+  const containerEl = document.createElement('pre');
+
+  const wrapperEl = document.createElement('div');
+  wrapperEl.innerHTML = "<h2>findDates</h2>" +
+    "from: " + new Date(Date.UTC(2020, 1 - 1, 1, 0, 0, 0)) + "<br />" +
+    "to: " + new Date(Date.UTC(2020, 1 - 1, 15, 23, 59, 59));
+  wrapperEl.insertAdjacentElement("beforeend", containerEl);
+
+  const perform = async () => {
+    const data = await layer.findDates(
+      bbox,
+      new Date(Date.UTC(2020, 1 - 1, 1, 0, 0, 0)),
+      new Date(Date.UTC(2020, 1 - 1, 15, 23, 59, 59)),
+      OrbitDirection.ASCENDING,
+    );
+
+    containerEl.innerHTML = "<ul>" + data.map(d => "<li>" + d + "</li>") + "</ul>";
```

Comment:
Nothing major, but this adds a comma (`,`) after each line, which looks a bit weird... this would fix it:
```
  containerEl.innerHTML = "<ul>" + data.map(d => "<li>" + d + "</li>").join("") + "</ul>";
```

Reply:
Thanks, fixed in all stories.